axelar-chains-config/info/testnet.json

```diff
@@ -1447,7 +1448,8 @@
       "approxFinalityWaitTime": 25,
       "onchainGasEstimate": {
         "chainName": "arbitrum",
```

Comment:
but it's different from `arbitrum-sepolia`, we wanted to support canonical name for testnet estimation as well.

Reply:
the canonical name is `arbitrum-sepolia` though. `arbitrum` refers to goerli, an outdated testnet. Devs provide the axelar registered chain name to the gas service and the gateway, otherwise, they're referring to the incorrect chain.

Reply:
So should I remove `chainName` from everywhere? I have it for optimism, base etc

Reply:
removed all